app/services/fact_parser.rb

```diff
@@ -179,10 +179,4 @@ def normalize_interfaces(interfaces)
   def not_implemented_error(method)
     "#{method} fact parsing not implemented in #{self.class}"
   end
-
-  def is_numeric?(string)
```

Comment:
I also scanned for it across all our repos (including katello) and found none which is why I just removed it. Just as a side note, the logic here doesn't actually match the function name - non-integer numeric values would return false. In addition, there are much better ways to check this w/o having to throw and catch exceptions.